generate-stackbrew-library.sh

```diff
@@ -5,7 +5,7 @@ shopt -s globstar
 declare -A aliases
 aliases=(
 	[9.2-jre7]='jre7'
-	[9.3-jre8]='latest jre8'
+	[9.4-jre8]='latest jre8'
```

Comment:
I think if we're not going to have the Official Images team build a `jetty:9.3` image anymore, then it should be removed entirely from the repo.

@gregw Following up on our discussion in #34, which version of Jetty do you think we should provide up-to-date images of? Will you guys still be doing releases on the 9.3 line?